### DISCUSSION TITLE
validate: { schema: ... } but for all args at once?

### DISCUSSION THREAD

**Discussion by babsonmatt** [hayes/pothos#349] [Ideas]
Is there any interest in being able to pass a schema to the validate all args method?  It doesn't seem like this is currently supported and probably unnecessary as you could just call zod.parse within the validate method yourself but it might be nice to keep consistent with the validate: { schema: ... } functionality supported by each arg individually.  Curious to see what everyone thinks!

**Comment by hayes** [comment]
Just pushed an update that will allow this, will be released soon

**Reply by babsonmatt** [reply]
Awesome, thank you!

**Comment by ricardo-valero** [comment]
does anyone have an example of this?

**Reply by hayes** [reply]
~https://github.com/hayes/pothos/blob/main/packages/plugin-validation/tests/example/schema/index.ts#L98~

**Reply by hayes** [reply]
Wrong link above, should have been https://github.com/hayes/pothos/blob/main/packages/plugin-validation/tests/example/schema/index.ts#L234-L245
